Update pricing.md (#1141)
flowforge/website: src/handbook/sales/pricing.md

Internal documents:
- [Price list with volume discounts](https://docs.google.com/spreadsheets/d/1q1OEo_5fr20txl__bJDR1cFrzv3qzvLLXcgEVFq6h68)

### Professional Services

> FlowFuse may offer support services as part of an annual subscription it it's deemed necessary to onboard the new customer or expand usage for a current customer. Any professional services or engineering resources provided will be subject to business case and CEO approval.

Reviewer: ```suggestion
FlowFuse may offer support services as part of an annual subscription if it's advantageous to onboard the new customer or expand usage for a current customer. Any professional services or engineering resources provided will be subject to business case and CEO approval.
```
